src/modules/user/user.service.ts

```diff
@@ -37,6 +37,20 @@ export class UserService {
     }
   }
 
+  // fetch all gamification data
+  async getAllGamified(): Promise<User[]> {
+    try {
+      const users = await this.userModel
+        .find({}, { _id: false })
+        .select('first_name last_name score photoUrl')
+        .sort({ score: 1 })
+        .exec();
+      return users;
+    } catch (e) {
+      throw new InternalServerErrorException(e);
+    }
+  }
+
```

Comment:
The frontend is using pagination and doesn't show all the info at the same time, please add pagination here

Reply:
@Abhishek-kumar09 Pagination Added